packages/gatsby/src/bootstrap/load-plugins/index.js

```diff
@@ -38,14 +38,11 @@ const flattenPlugins = plugins => {
 }
 
 module.exports = async (config = {}, rootDir = null, latestAPIs = {}) => {
```

Comment:
latestAPIs seems to be unused
```suggestion
module.exports = async (config = {}, rootDir = null) => {
```